### DISCUSSION TITLE
Production deployment - documentation for AWS

### DISCUSSION THREAD

**Discussion by satyagofylo** [langflow-ai/langflow#1105] [General]
Hi looking for production deployment documentation for AWS or similar cloud tools do let me know if any one deployed in prod, it would be helpful

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
